hat/assets/js/apps/Iaso/domains/app/translations/en.json

```diff
@@ -793,42 +793,42 @@
     "iaso.pages.title": "Pages",
     "iaso.periods.end": "End period",
     "iaso.periods.start": "Start period",
-    "iaso.permission.reports": "Reports",
-    "iaso.permissions.assignments": "Assignments",
-    "iaso.permissions.completeness": "Data completeness",
-    "iaso.permissions.completeness_stats": "Completeness stats",
-    "iaso.permissions.dataTasks": "Tasks",
-    "iaso.permissions.entities": "Beneficiaries",
-    "iaso.permissions.forms": "Forms",
+    "iaso.permission.reports": "Devices",
+    "iaso.permissions.assignments": "Attributions",
+    "iaso.permissions.completeness": "Export completeness",
+    "iaso.permissions.completeness_stats": "Completeness statistics",
+    "iaso.permissions.dataTasks": "Batch monitoring",
+    "iaso.permissions.entities": "Entities",
+    "iaso.permissions.forms": "Forms management",
     "iaso.permissions.iaso_dhis2_link": "Link with DHIS2",
-    "iaso.permissions.iaso_entity_duplicates_read": "See entity duplicates",
-    "iaso.permissions.iaso_entity_duplicates_write": "Edit entity duplicates",
-    "iaso.permissions.iaso_entity_type_write": "Beneficiary type create/update/delete",
-    "iaso.permissions.iaso_org_unit_groups": "Org unit groups",
-    "iaso.permissions.iaso_org_unit_types": "Org unit types",
-    "iaso.permissions.iaso_page_write": "Edit page",
+    "iaso.permissions.iaso_entity_duplicates_read": "Entity duplicates - Read only",
+    "iaso.permissions.iaso_entity_duplicates_write": "Entity duplicates - Read and Write",
```

Comment:
Are we sure that the write permission always gives access to the functionalities of the read permission?

Reply:
@quang-le it  is from the ticket Laure created.
![Screenshot from 2023-09-22 15-51-17](https://github.com/BLSQ/iaso/assets/19631540/cac2d19a-e00e-496e-95b1-456aa98fa323)